src/pages/ArticlePage.jsx

```diff
@@ -0,0 +1,46 @@
+import { useParams } from 'react-router-dom';
+import { ArticleFull } from '../components/ArticleFull';
+import { CommentsComp } from '../components/CommentsComp';
+import { useEffect, useState } from 'react';
+import { getArticleById } from '../utils/api';
+import { VotesComp } from '../components/VotesComp';
+import formatDateTime from '../utils/formatDateTime';
+
+export const ArticlePage = ({
+  isLoading,
+  setIsLoading,
+  isError,
+  setIsError,
+}) => {
+  const [article, setArticle] = useState({});
+  const [dateAndTime, setDateAndTime] = useState({});
+  const { article_id } = useParams();
+
+  useEffect(() => {
```

Comment:
Here 'isLoading' needs to be set to true, as last time, when all the articles loaded, it was set to false.